Doc/reference/expressions.rst

```diff
@@ -1255,10 +1263,10 @@ The ``%`` (modulo) operator yields the remainder from the division of the first
 argument by the second.  The numeric arguments are first converted to a common
 type.  A zero right argument raises the :exc:`ZeroDivisionError` exception.  The
 arguments may be floating point numbers, e.g., ``3.14%0.7`` equals ``0.34``
-(since ``3.14`` equals ``4*0.7 + 0.34``.)  The modulo operator always yields a
-result with the same sign as its second operand (or zero); the absolute value of
-the result is strictly smaller than the absolute value of the second operand
-[#]_.
+(since ``3.14`` equals ``4*0.7 + 0.34``.)  The modulo operator, unless overridden
+with the special :meth:`__mod__` method, always yields a result with the same sign
+as its second operand (or zero); the absolute value of the result is strictly
+smaller than the absolute value of the second operand [#]_.
```

Comment:
Again, I feel like mixing basic information with the ability to override with a special method, makes the entire thing less readable.